src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -6,20 +6,22 @@ topic: Develop
 tags: [add-ons, beginner, guide, mobile, webextensions]
 contributors:
   [caitmuenster, rebloor, juraj, mdnwebdocs-bot, ExE-Boss, Ding-Fan, andrewtruongmoz]
-last_updated_by: rebloor
-date: 2019-08-04
+last_updated_by: caitmuenster
+date: 2020-08-25
 ---
 
 {% capture page_hero_banner_content %}
 
 # Developing extensions for Firefox for Android
 
 ::: note alert
-In August 2020, Mozilla will begin rolling out a new, reimagined Firefox for Android experience. The  browser for Android has been rebuilt from the ground up using [GeckoView](https://mozilla.github.io/geckoview/), Mozilla's mobile browsing engine.
+In August 2020, Mozilla will begin rolling out a new, reimagined Firefox for Android experience (codenamed "Fenix"). The  browser for Android has been rebuilt from the ground up using [GeckoView](https://mozilla.github.io/geckoview/), Mozilla's mobile browsing engine.
 
-At the time of launch, only a limited number of [Recommended Extensions](https://support.mozilla.org/en-US/kb/recommended-extensions-program?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=developing-extensions-for-firefox-for-android) will be supported. We would like to support more extensions and we are currently evaluating how to best do that. New announcements around mobile extension support will be posted on the [Add-ons Blog](https://blog.mozilla.org/addons/category/mobile?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=developing-extensions-for-firefox-for-android).
+At the time of launch, only a limited number of [Recommended Extensions](https://support.mozilla.org/en-US/kb/recommended-extensions-program?utm_source=extensionworkshop.com&utm_medium=dev-article&utm_content=developing-extensions-for-firefox-for-android) will be supported. We would like to support more extensions and we are currently evaluating how to best do that.
 
-This article describes the development process of extensions on Firefox for Android through version 68. These methods are not supported for versions 69+.
+Currently, arbitrary extensions can only be loaded temporarily in Fenix. We are working on supporting a pref on [Firefox for Android Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) to enable persistent loading for arbitrary extensions. You can follow progress by subscribing to [this issue](https://github.com/mozilla-mobile/fenix/issues/14034).
```

Comment:
Nit, are we sure that it is going to be a pref? maybe we should call it "a setting" (which will clarify then later once the implementation details for that plan are detailed enough), how that sounds?